CHANGELOG.md

```diff
@@ -1,9 +1,30 @@
-# CyberArk AWS Cloud Package Release Notes
+# CyberArk PAMonCloud Offering Release Notes
+The PAMonCloud Offering includes CyberArk PAM products, delivered as AWS AMIs and Azure images, along with AWS CloudFormation and Azure ARM templates to automate deployment.
 
-The AWS Cloud package includes CyberArk PAS products, delivered as AMIs and AWS CloudFormation templates to automate deployment.
 
+## [PAMonCloud v14.4] (TBD)
 
-## [PAS on Cloud v14.2] (18.6.2024)
+### Added
+- #### Azure : 
+    - ARM templates now support Azure GovCloud accounts.
+- #### AWS : 
+    - Product configuration failures are tracked and raised in real-time, rather than only at stack timeout.
+- Windows Server 2022 images are now available for all Windows based components.
```

Comment:
tab missing

Reply:
intentional (these are not AWS specific changes)